reference/promotions.v3.yml

```
@@ -196,6 +196,72 @@ paths:
         '204':
           description: The deletion was successful or the specified resource does not exist.
           content: {}
+
+  '/promotions/{promotion_id}/codegen':
+    post:
+      tags:
+        - Coupon Codes (Bulk)
+      summary: Generate Bulk Coupon Codes
+      description: |-
+        Generate a batch of coupon codes for a particular bulk coupon promotion.
+        
+        **Note:**
+        * batch_size is limited to 250 codes per request for first version. If batch_size is not an integer or larger than 250, it will return a 422 error code.
```

🍹 for external devs, they may not be aware of what the "first version" means 

```suggestion
        * batch_size (number of codes generated per request) is limited to 250. If batch_size is not an integer or larger than 250, it will return a 422 error code.
```